src/app/helptext/system/general.ts

```diff
@@ -188,5 +188,10 @@ export const helptext_system_general = {
   snackbar_download_success: {
     title: T("Download Sucessful"),
     action: T("Success")
+  },
+
+  validation_errors: {
+    ui_address: T("When 0.0.0.0 has been selected, selection of other addresses is not allowed."),
```

Comment:
I don't know if this works as a general error.  Does selecting 0.0.0.0 or :: always mean "all addresses"?  If it does, we should say so: "When all has been selected by entering <samp>0.0.0.0</samp> or <samp>::</samp>, no other addresses can be entered."

Reply:
I believe so, you can either pick specific ip addresses or the 0.0.0.0 or :: wildcards

Reply:
Okay, how about:
"When all addresses have been selected by choosing <samp>0.0.0.0</samp> or <samp>::</samp>, additional addresses cannot be chosen."